financial-connections/src/main/java/com/stripe/android/financialconnections/domain/GoNext.kt

```diff
@@ -1,34 +1,12 @@
 package com.stripe.android.financialconnections.domain
 
-import com.stripe.android.core.Logger
 import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest.Pane
 import com.stripe.android.financialconnections.navigation.NavigationCommand
 import com.stripe.android.financialconnections.navigation.NavigationDirections
-import com.stripe.android.financialconnections.navigation.NavigationManager
-import javax.inject.Inject
-
-/**
- * Navigates to the next screen given a [Pane].
- */
-internal class GoNext @Inject constructor(
```

Comment:
Nit: Should this file be renamed following the removal of the class?

Reply:
https://github.com/stripe/stripe-android/pull/6968/commits/d51d38be17ebd1f551a41ba9d1cd5ca0f3869d4c